rust/platform/triple.bzl

```diff
@@ -59,6 +59,9 @@ def triple(triple):
         system = "android"
         abi = "eabi"
 
+    if system == "darwin":
```

Comment:
I have not handled the case where someone can specify `"os": "darwin"` in their `target_json`: https://github.com/bazelbuild/rules_rust/blob/main/rust/toolchain.bzl#L654-L655

Reply:
I think that this should just be allowed. Using `target_json` I think is uncommon but we should allow for whatever anyone puts in those files. E.g. not map `darwin` to `macos`. That's a user configuration error if that's what they have. But parsing `aarch64_apple_darwin` to be target_os = "macos"` is totally appropriate.